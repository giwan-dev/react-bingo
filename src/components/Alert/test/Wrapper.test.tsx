import AlertWrapper from '../Wrapper';
import { shallow } from 'enzyme';
import React from 'react';

describe('AlertWrapper component', () => {
  it('should render withount crashing', () => {
    const wrapper = shallow((
      <AlertWrapper />
    ));

    expect(wrapper).toMatchSnapshot();
  });
});
