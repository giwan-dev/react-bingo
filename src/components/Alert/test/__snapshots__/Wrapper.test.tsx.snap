// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertWrapper component should render withount crashing 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: absolute;
  top: 50%;
  left: 50%;
  margin: auto;
  width: 500px;
  height: 80px;
  border-radius: 5px;
  padding: 20px;
  background-color: white;
  transform: translate(-50%, -50%);
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
